apps/rxjs.dev/package.json

```diff
@@ -77,9 +77,10 @@
     "@types/jasmine": "~3.6.0",
     "@types/jasminewd2": "^2.0.3",
     "@types/lunr": "^2.3.3",
-    "@types/node": "^20.6.3",
+    "@types/node": "^20.11.0",
     "@types/svgo": "^1.3.3",
     "@types/trusted-types": "^2.0.7",
+    "@types/yamljs": "^0.2.34",
```

Comment:
Ran into issues with TypeScript building the dev app and had to add some dev deps and update types in a few files.